docs/reference-guides/interactivity-api/core-concepts/how-the-interactivity-api-works

```diff
@@ -0,0 +1,276 @@
+            <span data-wp-text="context.closedItemMessage"></span>
+            <p class='accordion__closed-item' data-wp-bind--hidden="!state.isLastItemSet">
+                <span data-wp-text="context.lastOpenItemMessage"></span>&nbsp;
+                <span class='accordion__closed-item-name' data-wp-text="state.lastOpenedItemName"></span>
+            </p>
+        </div>
+    </div>
+
+    <div class='accordion__item' data-wp-context='{"isItemClosed":true,"itemName":"First"}'>
+        <p class='accordion__item-title' data-wp-on--click="actions.toggleItem" data-wp-text="context.itemName"></p>
+        <div class='accordion__item-content' data-wp-bind--hidden="context.isItemClosed">Content of the first item</div>
+    </div>
+
+    <div class='accordion__item' data-wp-context='{"isItemClosed":true, "itemName":"Second"}'>
+        <p class='accordion__item-title' data-wp-on--click="actions.toggleItem" data-wp-text="context.itemName"></p>
+        <div class='accordion__item-content' data-wp-bind--hidden="context.isItemClosed">Content of the second item
+        </div>
+    </div>
+</div> 
+```
+
+Note: The `wp_interactivity_state` and `wp_interactivity_data_wp_context` calls are optional,
+while `data-wp-interactive="accordion"` is a required directive to tell the Interactivity API that we want this element and its nested DOM elements to be recognized by the Interactivity API.
+
+The following will happen:
+
+#### 2.1) The `wp_interactivity_state` call initializes some server-side state for the block.
+
+It's stored in a global variable that keeps the state of all blocks throughout the request.
+
+#### 2.2) The `wp_interactivity_data_wp_context` call turns the passed variables into a data attribute with JSON.
+
+In this example, the defined strings will be translated (if needed) and turned into the `data-wp-context` attribute:
+
+```html
+data-wp-context='{"openItemMessage":"Current open item is","closedItemMessage":"Items are closed","lastOpenItemMessage":"Last opened item is"}
+```
+
+#### 2.3) WordPress, upon encountering `data-wp-interactive="accordion"`, processes the block directives.
+
+In this case, there are 7 directives to be processed: 2 `data-wp-bind--hidden`, 5 `data-wp-text`
+and `data-wp-class--dark`. Using the HTML
+API and defined data (state in the global block-related PHP variable and variables as JSON attributes), WordPress will
+process them and change the markup.
+
+As a result, the span will contain the translated text and the hidden attribute
+will be added to the matching (first) item. The `data-wp-class--dark` directive will be skipped, as no such state is
+available. This is a client-only state, which will be defined on the client side.
+
+### 3. Scripts enqueuing
+
+#### 3.1) WP automatically adds the `interactivity.min.js` script alias.
+
+Besides adding the `<script>` tags for all the
+assets used on the page, WP will automatically add the `@wordpress/interactivity` alias, pointing to
```

Comment:
It should be clarified what this "alias" is and where it comes from. It's an entry in the [import map](https://make.wordpress.org/core/2024/03/04/script-modules-in-6-5/) that is added by WordPress